kubernetes_asyncio/watch/watch_test.py

```diff
@@ -22,6 +22,38 @@
 
 class WatchTest(TestCase):
 
+    def test_find_return_type(self):
+        """A few basic test cases to ensure it does what it says."""
+        fun = kubernetes_asyncio.watch.watch._find_return_type
+
+        # Must return None because the `None` object has an empty doc string
+        # and is not a K8s type.
+        assert fun(None) is None
```

Comment:
Using assert methods from TestCase (eg. assertIsNone, assertEqual) will be more consistent.